packages/devtools_app/lib/src/screens/performance/panes/flutter_frames/flutter_frames_chart.dart

```diff
@@ -901,6 +901,7 @@ class ChartAxisPainter extends CustomPainter {
             baselineAdjust,
       ),
     );
+    textPainter.dispose();
```

Comment:
nit use cascades: 
```
textPainter
  ..paint(...)
  ..dispose()
```

Reply:
this could even be combined with the ..layout() cascade above

Reply:
Done.